mix.exs

```diff
@@ -32,14 +39,27 @@ defmodule ExForce.Mixfile do
 
   defp deps do
     [
-      {:tesla, "~> 1.3"},
+      {:tesla, "~> 1.3", optional: true},
       {:jason, "~> 1.0"},
-      {:bypass, "~> 1.0", only: :test},
-      {:credo, "~> 1.1", only: :dev, runtime: false},
-      {:dialyxir, "~> 0.5", only: :dev, runtime: false},
-      {:excoveralls, "~> 0.12", only: :test},
+      # Docs
+      {:inch_ex, "~> 2.0.0", only: :dev},
       {:ex_doc, "~> 0.21", only: :dev, runtime: false},
-      {:inch_ex, "~> 2.0.0", only: :dev}
+      # Lint
+      {:credo, "~> 1.1", only: [:dev, :test], runtime: false},
+      {:dialyxir, "~> 0.5", only: [:dev, :test], runtime: false},
+      # Testing
+      {:bypass, "~> 1.0", only: :test},
+      {:excoveralls, "~> 0.12", only: :test}
+    ]
+  end
+
+  defp aliases do
```

Comment:
👍  for having `test.all` 

but let's move aliases and deps reorg into separate MR so that we can merge it first!

Reply:
Alright 👍 

Done in ae9e5cc4794e1e63cd6fd35a2d49659cdda261b6